File: Level1/Budget/Solution.js
function solution(d, budget) {
    var answer = 0;
    d.sort(function s(a,b) {
        return a - b;
    })
    let i = 0;
    while ((budget - d[i]) >= 0 && i < d.length) {
        budget -= d[i];
        i++
        answer++;
    }
    return answer;

}

File: Level1/Budget/Solution.java
package Budget;
import java.util.*;
public class Solution {
    public int solution(int[] d, int budget) {
        int i = 0;
        int answer = 0;
        Arrays.sort(d);
        while (i < d.length && budget - d[i] >= 0) {
            budget -= d[i];
            i++;
            answer++;
        }
        return answer;
    }
}